uiautomationutilities/perf_tool/ReportHelper.cs

```diff
@@ -0,0 +1,223 @@
+﻿//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Mail;
+using System.Net.Mime;
+using System.Text;
+
+namespace PerfClTool
+{
+    internal class ReportHelper
+    {
+        public struct TableResultSummary
+        {
+            public int PassCount;
+            public int FailCount;
+        }
+
+        /// <summary>
```

Comment:
This function is documented... great! Can we (piecemeal) take steps to do this across the other classes comprising this perf measurement tool? Our team is going to be responsible for owning this implementation for the foreseeable future, it would be great to have it well documented with cleanly formatted source, etc. Incl. removing broken/commented out/inaccessible code